src/engine/main.cpp

```diff
@@ -911,6 +912,7 @@ int main(int argc, char **argv)
     #endif
     #endif
 
+    setlocale( LC_ALL, "" );
```

Comment:
I am absolutely no fan of such code. It is very likely to interfere with our localization plans.

Why is this needed? It's not even documented...

Reply:
to be completely honest, I am not sure if it is NEEDED, I just thought it couldn't hurt anybody. I'll remove it and see if it still works